bin/add-alb.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+if [[ -z $1 ]]
```

Comment:
this might be a job for getopts

Reply:
also, quote your variables

Reply:
Agreed - though I figured that bash might be more accessible to Yale's DevOps team.  So if they're going to maintain this long-run probably shell scripting; whereas, if Software engineering is going to own this, then ruby makes total sense (in honesty, it would have been faster to do in ruby and I could have gotten even better error detection in)